crypto/src/test/java/org/web3j/crypto/TransactionDecoderTest.java

```diff
@@ -13,15 +13,14 @@
 package org.web3j.crypto;
 
 import java.math.BigInteger;
+import java.security.SignatureException;
 
 import org.junit.jupiter.api.Test;
 
+import org.web3j.crypto.transaction.type.Transaction1559;
 import org.web3j.utils.Numeric;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertNull;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Wildcard imports are not used in the Web3j code style.

Reply:
Done.